config/database.yml.example

```diff
@@ -1,85 +0,0 @@
-# PostgreSQL. Versions 8.2 and up are supported.
```

Comment:
This file should not be deleted.  It is intended to serve as an example to all future developers.